content/Classes/Classes.q.md

```diff
@@ -0,0 +1,1088 @@
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=SS, answer=[4]
+
+On line 12, what does the statement `$myObject = new MyClass;` do?
+
+ - It sets the value of `$myObject` to `MyClass`.
+
+ - It replaces the value of `$myObject` with `MyClass`.
+
+ - It accesses the value of `MyClass` through `$myObject`.
+
+ - It creates the `$myObject` object of the `MyClass` class.
+
+ - It assigns the value of the `MyClass` class to `$myObject`.
+
+
+/// type=SS, answer=[5]
+
+On line 14, what does the statement `echo $myObject->myProp;` do?
+
+ - It creates the `myProp` property of `$myObject`.
+
+ - It sets the value of the `myProp` property of `$myObject`.
+
+ - It returns the value of the `myProp` property of `$myObject`.
+
+ - It removes the value of the `myProp` property of `$myObject`.
+
+ - It displays the value of the `myProp` property of `$myObject`.
+
+
+:::
+
+
+:::
+
+/// type=REPL
+
+```php
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
+    	
+    $myObject = new MyClass;
+
+    echo $myObject->myProp;
+?>
+```
+/// type=SS, answer=[4]
+
+On line 14, replace the statement `echo $myObject->myProp;` with `$myObject->myMethod();`. Execute the program. What is its output?
+
+ - It prints `MyClass`.
+
+ - It produces an error.
+
+ - No output is displayed.
+
+ - It prints `This is a class method.`.
+
+ - It prints `This is a class property.`.
+ 
+:::
+
+
+:::
+
+/// type=REPL
+
+```php
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
+    	
+    $myObject = new MyClass;
+
+    $myObject->myMethod();
+?>
+```
+/// type=SS, answer=[1]
+
+On line 14, what does the statement `$myObject->myMethod();` do?
+
+ - It calls the `myMethod()` method of `$myObject`.
+
+ - It checks the `myMethod()` method of `$myObject`.
+
+ - It creates the `myMethod()` method of `$myObject`.
+
+ - It removes the `myMethod()` method of `$myObject`.
+
+ - It evaluates the `myMethod()` method of `$myObject`.
+
+:::
+
+
++++
+
+
++++
+
+### Part 2: Knowledge Assessment
+
+/// type=SS, answer=[5]
+
+Which statements are true about a class?
+
+ - It is an instance of an object.
+
+ - It is a basic building block of a method.
+
+ - It is an actual representation of a function.
+
+ - It is a function that performs a specific action.
+
+ - It is a blueprint that defines the characteristics and behaviors of all objects of a specific kind.
+
+
+/// type=SS, answer=[1]
+
+Which statement best describes the `class` keyword?
+
+ - It creates a class.
+
+ - It accesses a class.
+
+ - It sets a property of a class.
+
+ - It defines a method of a class.
+
+ - It removes a method of a class.
+
+
+/// type=SS, answer=[3]
+
```

Comment:
Sorry, we cannot enclose property, class, and methods with backticks here since they are not keywords/reserve words or commands in PHP. Only expected outputs, reserved words, variable names and etc are enclosed in backtick. Please check. Thanks.